pages/vi/profiles/Huralain.md

```diff
@@ -0,0 +1,20 @@
+# Hello Everyone!
+
+My name is **Huralain Mohamed** and I am located in **El Paso, TX, USA**. My current timezone is **MT(-6): Mountain Time**. I am currently operating on a MBP - **macOS Mojave version 10.14.6**.
+
+Just so you can get to know a little bit about me, I am 26 years old and a US Army veteran. I have been working in the tech industry for 9 years, however, I started learning how to code this year. I am excited to start working with you all at OLE to further expand my skillset.
+
+Some fun facts about me are:
++ Pizza is my favorite food.
++ I've been to 15+ countries.
++ I was born in LA, but raised in Boston.
++ I'm a fashion designer as well as a coder.
```

Comment:
Your "Fun facts" section's list isn't rendering correctly. 

Other than that. Good job!

Reply:
Thank you so much Caleb! I'll fix that issue :)